front/src/NotificationWindow.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color : ${props => props.backgroundColor}
+  border-style: solid;
+  border-size: 4px;
+  width: 90%;
+  position: relative;
+  margin-left:auto;
+  margin-right:auto;
+`;
+
+const NotificationHeader = styled.p`
+  text-align: center;
+  font-size: 5vmin;
+`;
+
+const NotificationText = styled.p`
+  padding-left: 10px;
+  padding-right: 10px;
+  text-align: center;
+  font-size: 8vmin;
+`;
+
+const NotificationCloseButton = styled.button`
+  border: none;
+  background-color: transparent;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 10vmin;
+  height: 10vmin;
+  &:focus {
+    outline: 0;
+  }
+  font-size: 2vmin;
+`;
+
+/**
+ * Creates a notification window
+ * Author: S.A. Tanja
+ *
+ * Props:
+ * - backgroundColor: sets the background color of the notification window
+ * - notificationType: sets the title of the notification window
+ * - onClick: on close function should be passed
+ * - children: the message of the notification window
+ */
+const NotificationWindow = props => {
+  return (
+    <Window backgroundColor={props.backgroundColor}>
+      <NotificationHeader>{props.notificationType}</NotificationHeader>
+      <NotificationCloseButton
+        onClick={() => {
+          props.onClick();
+        }}
+      >
+        <FontAwesomeIcon icon={faTimes} color="black" size="4x" />
+      </NotificationCloseButton>
+      <NotificationText>{props.children}</NotificationText>
```

Comment:
You pass the header text as a prop and the content as `children`. Do you expect to render anything else rather than simply text? If yes - it is fine, otherwise we dont need this.

Reply:
I thought it could be useful. Let's say the battery of a turtle is running low, then it can be possible to pass a button with a stop function for that specific turtle, so you don't have to look for the turtle first and then press the stop button.

Reply:
Then it is not `NotificationText` and not a paragraph.